src/components/DateInput.jsx

```diff
@@ -19,6 +19,8 @@ const propTypes = forbidExtraProps({
   onFocus: PropTypes.func,
   onKeyDownShiftTab: PropTypes.func,
   onKeyDownTab: PropTypes.func,
+
+  isFocused: PropTypes.bool, // describes actual DOM focus
```

Comment:
A little confused about the distinction between this bool and `focused` by name alone. Seems like it'd be better as something like `focusInputDOM` or something that makes it more explicit.